src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -947,12 +973,6 @@ def update_connected_cluster(cmd, client, resource_group_name, cluster_name, htt
         patch_cc_response = update_connected_cluster_internal(client, resource_group_name, cluster_name, tags, distribution, distribution_version, azure_hybrid_benefit)
 
     proxy_params_unset = (https_proxy == "" and http_proxy == "" and no_proxy == "" and proxy_cert == "" and not disable_proxy)
-
-    # Returning the ARM response if only AHB is being updated
```

Comment:
why is this check being removed? we should not do it since AKS HCI customers will see errors while enabling AHB on their clusters. (since they dont have LoS to the cluster)

Reply:
I didn't remove this. It's probably because I started this branch sometime back when your change wasn't pushed yet. Adding it